packages/core/src/Subscribe.tsx

```diff
@@ -12,15 +17,20 @@ import { Observable } from "rxjs"
  */
 export const Subscribe: React.FC<{
   source$: Observable<any>
-  fallback?: null | JSX.Element
+  fallback?: NonNullable<ReactNode> | null
 }> = ({ source$, children, fallback }) => {
   const [mounted, setMounted] = useState(0)
-  useEffect(() => {
-    const subscription = source$.subscribe()
+  useLayoutEffect(() => {
+    const subscription = source$.subscribe(noop, (e) =>
+      setMounted(() => {
+        throw e
+      }),
+    )
```

Comment:
Brings up the question: what should happen on complete?

Reply:
Nothing, really. Also, most of the times won't even happen, because the streams returned by `bind` don't complete.